### PR TITLE
[Fix] type of "counts" in COCO’s compressed RLE

### PR DESCRIPTION
**Pull request by Daa98**
## Motivation

"counts" in COCO’s compressed RLE format has type of 'str' or 'list'.
ea6e2d0d707f54058f5ecc31b0b2385

### CODE REVIEW

**Comment by Daa98** [comment]
Sorry, I  don't know how to resume the former pr #9000.

**Comment by ZwwWayne** [comment]
Hi @Daa98 ,
Thanks for your kind PR. You can simply push a commit to the origin branch, i.e., patch-1 in #9000 to continue it. It is OK if you want to modify in this PR.

File: mmdet/datasets/transforms/loading.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ def _process_masks(self, results: dict) -> list:
             elif isinstance(gt_mask, dict) and \
                     not (gt_mask.get('counts') is not None and
                          gt_mask.get('size') is not None and
-                         isinstance(gt_mask['counts'], list)):
+                         isinstance(gt_mask['counts'], (list, str))):
```
[the remark this answers is not in the document]
Comment (Daa98) [reply mmdet/datasets/transforms/loading.py:351]: "counts" can be in 'str' format and mmdet supports it.     https://github.com/open-mmlab/mmdetection/blob/6bd0ae907293eba06cc7e8c962056591c9808ac6/mmdet/datasets/transforms/loading.py#L291

For example
```
import pycocotools.mask as maskUtils
rle = {"counts": "acWd1=j`2<D3N1N3M2O1N3M2O2M2N2O2N12O1N2N2O1N2O1N2N2O1N2N2M3M3M3M3M3M3M3M3M[b[g4", "size": [2600, 2600]}
mask = maskUtils.decode(rle)
mask
```

**Review by BIGWangYuDong — APPROVED** [review]
LGTM

**Pull request merged**